keel-sql/src/main/resources/db/changelog/20201219-add-artifact-unique-index.yml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+- changeSet:
+    id: add-artifact-unique-index
+    author: fletch
+    changes:
+    - dropIndex:
+        tableName: delivery_artifact
+        indexName: delivery_artifact_byref_idx
```

Comment:
I don't think this index should be dropped, it's doing what we want `VerificationContext.artifactUid` to do, indexing on `delivery_config_name` and `reference`. Here's the original: https://github.com/spinnaker/keel/blob/master/keel-sql/src/main/resources/db/changelog/20200127-environment-artifact-pin-table.yml#L49-L56

Reply:
It just needs to be `unique: true`. I updated the PR.